lithops/standalone/standalone.py

```diff
@@ -264,6 +318,10 @@ def create_workers(workers_to_create):
 
         self.jobs.append(job_payload['job_key'])
 
+        # wait_workers_ready(total_required_workers)
+        threading.Thread(target=wait_workers_ready, args=(total_required_workers,), daemon=True).start()
```

Comment:
Here you are passing the total_required_workers, and this will potentially make all jobs to fail due the same reason I explained above: in our case (paid account), Lithops is unable to start all the total_required_workers due to Quota Limit, and it has to proceed with those workers that it has been able to create/start/find-in-the-master.

Reply:
Fixed, replaced to track new_workers list

Reply:
Fixed using the new workers_policy property in the standalone config